ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -716,6 +716,12 @@ const NetworksForm = ({
     !rpcUrl ||
     !chainId ||
     !ticker;
+  let displayRpcUrl = rpcUrl?.includes(`/v3/${infuraProjectId}`)
+    ? rpcUrl.replace(`/v3/${infuraProjectId}`, '')
```

Comment:
Is the `includes` needed if we're only doing a `replace` since the `replace` won't do anything if it's not included?

Reply:
Valid point, this was actually existing code which I did not changed.